### DISCUSSION TITLE
New lidl Silvercrest 3 gang switch, with 4 USB only recognize 1 switch

### DISCUSSION THREAD

**Discussion by steve41200** [Koenkk/zigbee2mqtt#11638] [Devices]
I bought a lidl Silvercrest 3 gang switch, with 4 USB  and it seems new version only recognize I1 who we can control

I see the switch but with OFF ? ON on I2 and I3. When I lick on it I see  Error Device '0x2c1165fffe2216a0' has no endpoint 'l2' in the log
I try directly in MQTT explorer to send payload but same error, I control only I1.

Have you a trick to make it working ?

**Comment by klarf-pixel** [comment]
Hi, I have the same problem, but my old Silvercrest 3 gang switch buy the last year work fine with control of the 3 switch separately.
It's a new hardware version or a bug ?
Thanks.

ps: the serial number of the last part : Model no HG06338-FR version 09/2021

**Comment by philosoma** [comment]
Was stuck as well at first, found answer here: https://www.zigbee2mqtt.io/devices/HG06338.html

Version I just got is 08/20/20
